deploy/charts/kaudit/kaudit-policy/default-policy.yaml

```diff
@@ -109,4 +109,19 @@
   combine-op: and
   conditions:
     - {field: verb, match-op: =, match-value: create}
-    - {field: resource-namespace, match-op: =, match-value: default}
\ No newline at end of file
+    - {field: resource-namespace, match-op: =, match-value: default}
+
+#
+# Enforcement Example
+#
+# - id: 200001
+#   name: enforce - deny exec into kube-system 
+#   status: true
+#   combine-op: and
+#   conditions:
+#     - {field: resource-type, match-op: =, match-value: pods}
+#     - {field: subresource, match-op: =, match-value: exec}
+#     - {field: resource-namespace, match-op: =, match-value: kube-system}
+#   enforcement: {} # Use {} to specify an indefinite enforcement rule
```

Comment:
I suggest you explicitly put here the 3 variations, otherwise it's not obvious where/how to put time-limit/count-limit
enforcement: {}
enforcement: {time-limit: 360000}
enforcement: {count-limit: 5}

Reply:
never mind , I figured out how you wrote it